src/platform/mhv/secondary-nav/containers/MhvSecondaryNav.jsx

```diff
@@ -28,7 +28,7 @@ export const mhvSecNavItems = [
     actionName: `${actionPrefix} - Messages`,
     icon: 'forum',
     href: `/my-health/secure-messages`,
-    appRootUrl: '/my-health/secure-messages',
+    appRootUrl: '/my-health/secure-messages/inbox',
```

Comment:
Change the appending of '/inbox' to href instead of appRootUrl.
```
href: `/my-health/secure-messages/inbox`,
appRootUrl: '/my-health/secure-messages',
```

Reply:
@nichia is `appRootUrl` field even needed in this prop? I am thinking about just leaving `href: `/my-health/secure-messages` without `/inbox` as we are handling redirect to `/inbox` on SM end anyway. is `appRootUrl` even needed here then?

Reply:
You're right, if `href` for secure messages is `/my-health/secure-messages`, which is the same as the `appRootUrl`, then we don''t need to keep the `appRootUrl` here

Reply:
@nichia pushed the update. THank you for the suggestions and the review. Would you mind re-reviewing please?